### PR TITLE
[GE-2024] add speaker

### PR DESCRIPTION
**Pull request by deniger**
[GE-2024] add speaker

### CODE REVIEW

File: static/events/2024-geneva/speakers/olivier-boder.jpeg
Comment (adrianmoisey) [inline static/events/2024-geneva/speakers/olivier-boder.jpeg:1]: Could I suggest a smaller JPEG?
This is 8MB, which will result in slower page loads

Comment (phrawzty) [reply static/events/2024-geneva/speakers/olivier-boder.jpeg:1]: @deniger Agreed with Adrian, this doesn't need to be print quality ;)

Comment (deniger) [reply static/events/2024-geneva/speakers/olivier-boder.jpeg:1]: yes sorry it's done ( png format). Sorry didn't check that before

**Comment by deniger** [comment]
@phrawzty sorry for this point. it's solved now

**Pull request merged**